app/views/shared/_flash.html.erb

```diff
@@ -8,7 +8,7 @@
   <div class="govuk-notification-banner__content">
     <% flash.each do |type, msg| %>
       <div class="flash <%= type %>">
-        <p><%= msg %></p>
+        <p class="govuk-notification-banner__heading"><%= msg.html_safe %></p>
```

Comment:
Just noticed this @Rich-Cooley 

I think you would be better declaring the header in the `en` file and removing this class.

This flash file affects all of the messages across the site. So all of the notes and contacts messages will now be bold.

```yml
      create:
        success:
          <p class="govuk-notification-banner__heading">Voluntary project successfully created<p>
          <p class="govuk-body">Add details of any contacts you have for the school, trust, local authority and solicitors.</p>
          <p class="govuk-body">A caseworker will be assigned this project.</p>
          <p class="govuk-body">There is nothing more for you to do.</p>
```

Reply:
My initial thought is yes, when a notification banner is shown then it should be in the larger text… but then I think if I were to look in the yml file it would be more readable which part was the header and which part was the body. Personally I think I'd like to keep it how it is, and add that `<p class="govuk-notification-banner__heading">` to all the notification banner messages.

Reply:
So they will go from this:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/47089130/206739640-be9ac0ca-b7dc-49f5-943b-786e7eb02987.png">

To this:
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/47089130/206739873-2e02c088-b2a4-4ec6-83e8-91a88de741ec.png">

Reply:
Ok I have sorted that as I'm not confident on finding all the error messages and this should give consistency like you suggested.